server/store/welcome_store.go

```diff
@@ -0,0 +1,36 @@
+package store
```

Comment:
Why did you choose not to embed this data in the User records?

Reply:
As spoken through zoom, this data (the id of the post where the connect link is) is created before the User even exist. Therefore, it cannot be embed in the User records.